app/models/user.rb

```diff
@@ -2,10 +2,17 @@ class User < ApplicationRecord
   has_many :user_videos, dependent: :destroy
   has_many :videos, through: :user_videos
 
+
+  has_many :user_followers
```

Comment:
Initially wanted to make followers and followees but the verbage was making things confusing so ended up going with 'friendships'